ray/README.md

```diff
@@ -1,68 +1,64 @@
-# Deploying Ray with Open Data Hub
+# Deploying Ray on Open Data Hub via the KubeRay Operator
 
-Integration of [Ray](https://docs.ray.io/en/latest/index.html) with Open Data Hub on OpenShift. The ray operator and other components are based on https://docs.ray.io/en/releases-1.13.0/cluster/kubernetes.html
+The code found here is a subset of https://github.com/ray-project/kuberay. Specifically, the manifests needed to enable Ray with an Open Data Hub deployment. 
 
-## Components  of the Ray deployment
 
-1. [Ray operator](./operator/base/ray-operator-deployment.yaml): The operator will process RayCluster resources and schedule ray head and worker pods based on requirements. Image Containerfile can be found [here](https://github.com/thoth-station/ray-operator/blob/master/Containerfile). 
-2. [Ray CR](./operator/base/ray-custom-resources.yaml):  RayCluster Custom Resource (CR) describes the desired state of ray cluster.
-3. [Ray Cluster](./cluster/base/ray-cluster.yaml): Defines an instance of an example Ray Cluster. Image Containerfile can be found [here](https://github.com/thoth-station/ray-ml-worker/blob/master/Containerfile) 
- 
+## Components  of the KubeRay deployment
 
-## Deploy the RayCluster Components:
+1. Namespace 'ray-system'
+2. Custom Resource Definitions
+3. Operator
+4. Roles and Service Accounts
 
-Prerequisite to install RayCluster with ODH:
+## Deploy KubeRay:
 
-* Cluster admin access
-* An ODH deployment
-* [Kustomize](https://kustomize.io/) 
+Make sure that you have the KubeRay kustomizeConfig in your ODH KfDef. This will ensure that all of the components above get installed as part of your ODH deployment.  
```

Comment:
@MichaelClifford I think it would be better to provide a valid kfdef in this repo under the `ray` directory. Then the instructions here become to oc create that kfdef file.

Reply:
I'm happy to to it that way, but the rest of the components in this repo provide only the KustomizeConfig snippet. And the valid KfDef with all the components is under the 'KfDef' directory. 

So that changes would introduce a different format for the Ray directory. 

@LaVLaS Do you mind if we make this sort of change? I don't care either way, but wanted to point it out.

Reply:
IMO, the README should have two instructions for deployment
1. Standalone deployment by running `kustomize` on the manifest in the repo
2. Instructions for deploying via the kfdef.  

If there are required ODH dependencies that Ray needs then I would say a kfdef file is required.